bigquery_magics/bigquery.py

```diff
@@ -114,10 +118,12 @@
 from google.cloud.bigquery.dataset import DatasetReference
 from google.cloud.bigquery.dbapi import _helpers
 from google.cloud.bigquery.job import QueryJobConfig
+import pandas
 
 from bigquery_magics import line_arg_parser as lap
 import bigquery_magics._versions_helpers
 import bigquery_magics.config
+from bigquery_magics.graph_server import GraphServer, convert_graph_data
```

Comment:
And here: https://google.github.io/styleguide/pyguide.html#22-imports

Aside: Does this introduce any required dependencies if we import this way? Perhaps we need a `try/except` block for the graph-related imports?